src/Json.php

```diff
@@ -57,27 +57,27 @@ public function getEndpoint()
         return $this->endpoint;
     }
 
-    public function get($path, array $params = [])
+    public function get($path, array $params = [], array $headers = [])
```

Comment:
What do you think of renaming  $headers to $options, and make 'headers' a optional key in options? I think it could make it more resilient to changes in the future, if we need any. As in:

```php
$client->get('/foo', [], [ 'headers' => ['foo' => 'bar'] ]);
```